src/app/home/event-user-list/event-user-list.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Input, ViewChild, Output, EventEmitter } from '@angular/core';
+import { ClrModal } from '@clr/angular';
+import { Progress } from 'src/app/data/progress';
+import { User } from 'src/app/data/user';
+
+interface userListUser extends User {
+  numberOfSessions?: number;
+  activeScenarioName?: string;
+}
+
+@Component({
+  selector: 'event-user-list',
+  templateUrl: './event-user-list.component.html'  
+})
+export class EventUserListComponent implements OnInit {
+
+  @ViewChild("userListModal") userListModal: ClrModal;
+
+  @Input()
+  public users: User[];
+
+  @Input()
+  public progress: Progress[]; 
+
+  @Input()
+  public filterName: Function;
+
+  @Output() nameClickedEvent = new EventEmitter<string>();
```

Comment:
I would go with the following
```suggestion
  @Output() userSelected = new EventEmitter<User>();
```

- drops the `Event` suffix
- abstracts the action from click to select
- uses a semantic type, not a plain string. So consumers know what to expect.

Of course, this might be all a matter of taste